mjidris/irondb
# US173: Add Error Page

## Objective
Add the error page.

## Detailed Description
Converted the error page to a react component. I honestly don't know how or when this page was ever used in the previous web app 🤷‍♂️, but hey it was good practice to get familiar with React.

## Screenshot
![image](https://user-images.githubusercontent.com/40260591/72955942-d21dc380-3d52-11ea-8bc6-8a200f77bd10.png)

## How to Test (if applicable)
1. Spin up the React app with `npm start` in the `client` folder.
2. Manually navigate to `http://localhost:8000/error`. 
3. Observe the error component appears & the title changes.

## Requirements
[Taiga User Story](https://tree.taiga.io/project/cbrobsto-iron-db/us/173)
[Taiga Task](https://tree.taiga.io/project/cbrobsto-iron-db/task/202)
